ui-components/dialog.md

```diff
@@ -184,6 +183,66 @@ See the [dialog configuration options](#configurationoptions) documentation for
 }
 ```
 
+##### Dialog menu buttons
+
+A dialog menu button is a split button that can be used to provide a drop-down list of items in a dialog footer.
+
+When dialog menu items are clicked, a [_dialog `onAction` callback_](#onactionfunction) is triggered. The `name` of the menu item is passed into the onAction callback.
+Clicking on the menu footer button won't trigger any callbacks and will only open the menu of specified items.
+
+###### Dialog menu button
+
+The following options can be specified for a dialog menu button:
+
+| Name    | Value                  | Requirement | Description                                                                    |
+| ------- | ---------------------- | ----------- | ------------------------------------------------------------------------------ |
+| items   | array                  | required    | An array of [dialog menu items](#dialogmenuitems).                             |
+| text    | string                 | optional    | Text to display if no icon is found.                                           |
+| icon    | string                 | optional    | Name of the icon to be displayed. Must correspond to an icon in the icon pack. |
+| tooltip | string                 | optional    | Text for button tooltip.                                                       |
+| onSetup | (api) => (api) => void | optional    | default: () => () => {} - Function that's invoked when the button is rendered. |
```

Comment:
Asked and this actually shouldn't be exposed sorry! So we shouldn't include it in the docs and we'll remove it in the source.

Reply:
Deleted